0001-emulator-Use-external-hw-common.h-from-LiteX.patch

```diff
@@ -0,0 +1,82 @@
+From f88b259ebacb84624f2df3cfd14db56a5dd2ca4f Mon Sep 17 00:00:00 2001
```

Comment:
Patching is just a temporary solution. There is a PR to VexRiscv pending with the same changes: https://github.com/SpinalHDL/VexRiscv/pull/115.